src/Symfony/Routing/ApiLoader.php

```diff
@@ -107,7 +107,7 @@ public function load($data, $type = null): RouteCollection
 
             foreach ($this->resourceMetadataFactory->create($resourceClass) as $resourceMetadata) {
                 foreach ($resourceMetadata->getOperations() as $operationName => $operation) {
-                    if ($operation->getRouteName()) {
+                    if ($operation->getRouteName() || ($operation instanceof HttpOperation && str_starts_with($operation->getUriTemplate(), '/.well-known/genid/'))) {
```

Comment:
I find this a little ugly, couldn't we have a special parameter in the operation instead?